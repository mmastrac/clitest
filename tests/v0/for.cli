#!/usr/bin/env clitest --v0

for x in 1 2 3 {
    $ printf "$x\n"
    *
}

set A 1;
set B 2;
set C 3;

for x in $A $B $C {
    set LAST $x;

    $ printf "$x\n"
    *
}

$ echo $LAST
! 3
