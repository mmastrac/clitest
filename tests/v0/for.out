Running <root>/for.cli ...

printf "$x\n"
---
1
---
✅ OK

printf "$x\n"
---
2
---
✅ OK

printf "$x\n"
---
3
---
✅ OK

set A 1

set B 2

set C 3

set LAST 1

printf "$x\n"
---
1
---
✅ OK

set LAST 2

printf "$x\n"
---
2
---
✅ OK

set LAST 3

printf "$x\n"
---
3
---
✅ OK

echo $LAST
---
3
---
✅ OK

<root>/for.cli PASSED
1 test(s) passed
